tokio/src/sync/semaphore.rs

```diff
@@ -210,6 +210,55 @@ use std::sync::Arc;
 /// }
 /// ```
 ///
+/// ## Ensure Tests with Mutual Dependencies Run Sequentially
```

Comment:
I think this wording is better here.
```suggestion
/// ## Prevent tests from running in parallel
```

Reply:
done.